code/game/gamemodes/changeling/changeling.dm

```diff
@@ -46,143 +31,34 @@ GLOBAL_LIST_INIT(possible_changeling_IDs, list("Alpha","Beta","Gamma","Delta","E
-		destroy_objective.find_target()
-		changeling.objectives += destroy_objective
-	else
-		var/datum/objective/assassinate/kill_objective = new
-		kill_objective.owner = changeling
-		kill_objective.find_target()
-		changeling.objectives += kill_objective
-
-		if(!(locate(/datum/objective/escape) in changeling.objectives))
-			var/datum/objective/escape/escape_with_identity/identity_theft = new
-			identity_theft.owner = changeling
-			identity_theft.target = kill_objective.target
-			if(identity_theft.target && identity_theft.target.current)
-				identity_theft.target_real_name = kill_objective.target.current.real_name //Whoops, forgot this.
-				var/mob/living/carbon/human/H = identity_theft.target.current
-				if(!HAS_TRAIT(H, TRAIT_GENELESS)) // For species that can't be absorbed - should default to an escape objective
-					identity_theft.explanation_text = "Escape on the shuttle or an escape pod with the identity of [identity_theft.target_real_name], the [identity_theft.target.assigned_role] while wearing [identity_theft.target.p_their()] identification card."
-					changeling.objectives += identity_theft
-				else
-					qdel(identity_theft)
-
-	if(!(locate(/datum/objective/escape) in changeling.objectives))
-		if(prob(70))
-			var/datum/objective/escape/escape_objective = new
-			escape_objective.owner = changeling
-			changeling.objectives += escape_objective
-		else
-			var/datum/objective/escape/escape_with_identity/identity_theft = new
-			identity_theft.owner = changeling
-			identity_theft.find_target()
-			changeling.objectives += identity_theft
-	return
-
-/datum/game_mode/proc/greet_changeling(datum/mind/changeling, you_are=1)
-	SEND_SOUND(changeling.current, sound('sound/ambience/antag/ling_aler.ogg'))
-	if(you_are)
-		to_chat(changeling.current, "<span class='danger'>You are a changeling!</span>")
-	to_chat(changeling.current, "<span class='danger'>Use say \":g message\" to communicate with your fellow changelings. Remember: you get all of their absorbed DNA if you absorb them.</span>")
-	to_chat(changeling.current, "<B>You must complete the following tasks:</B>")
-	if(changeling.current.mind)
-		if(changeling.current.mind.assigned_role == "Clown")
-			to_chat(changeling.current, "You have evolved beyond your clownish nature, allowing you to wield weapons without harming yourself.")
-			changeling.current.dna.SetSEState(GLOB.clumsyblock, FALSE)
-			singlemutcheck(changeling.current, GLOB.clumsyblock, MUTCHK_FORCED)
-			var/datum/action/innate/toggle_clumsy/A = new
-			A.Grant(changeling.current)
-	var/obj_count = 1
-	for(var/datum/objective/objective in changeling.objectives)
-		to_chat(changeling.current, "<B>Objective #[obj_count]</B>: [objective.explanation_text]")
-		obj_count++
-	to_chat(changeling.current, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Changeling)</span>")
-	return
+	changeling_amount = 1 + round(num_players() / 10)
```

Comment:
I feel like this `10` could be a config option, however that should probably come later.